src/analyze/analyze.c

```diff
@@ -1802,6 +1804,169 @@ static int dump_syscall_filters(int argc, char *argv[], void *userdata) {
 }
 #endif
 
+#if HAVE_LIBBPF
```

Comment:
there's no usage of libbpf here? So you can reduce the scope of the `ifdef` a lot, maybe just an early return in `dump_filesystems`